extractor/intermediate.scala

```diff
@@ -91,7 +91,14 @@ case class API(
       else fixpoint(newInUse)
     }
 
-    val inUseNames = fixpoint(modelsInUse)
+    val recursivelyUsedModels = fixpoint(modelsInUse)
+
+    // check models forcibly included are not already used by the routes
+    val modelsIntersection = recursivelyUsedModels.intersect(modelsForciblyInUse)
+    if (!modelsIntersection.isEmpty)
+      throw new Exception(s"The following models are already used by the routes, no need to force inclusion: $modelsIntersection")
```

Comment:
Btw, the output is redirected to `> /dev/null` from `npm-metarpheus.sh` - I think we can solve this problem quiet easily. However, I'd open a new issue for this.